pkg/datapath/gneigh/cells.go

```diff
@@ -28,10 +28,10 @@ func (def Config) Enabled() bool {
 }
 
 func (def Config) Flags(flags *pflag.FlagSet) {
-	flags.String(L2PodAnnouncementsInterface, def.L2PodAnnouncementsInterface, "Interface used for sending gratuitous arp messages")
+	flags.String(L2PodAnnouncementsInterface, def.L2PodAnnouncementsInterface, "Interface used for sending gratuitous arp and nd messages")
```

Comment:
Nit: why not fix the case?

```suggestion
	flags.String(L2PodAnnouncementsInterface, def.L2PodAnnouncementsInterface, "Interface used for sending gratuitous ARP and ND messages")
```

Reply:
Done, and also used `NDP` instead of `ND` for consistency.